www/src/components/landing/navbar/navbar.astro

```diff
@@ -7,11 +7,21 @@ import GithubStarButton from "./githubStarButton.astro";
   class="z-[1] flex p-[0.5%] xl:p-[1%] 2xl:p-[1.5%] text-white justify-between items-center w-full"
 >
   <div class="m-2 h-10 w-10"></div>
+
```

Comment:
@todevmilen can you remove this extra space?

Reply:
Done :)